lib/jxl/dec_group.cc

```diff
@@ -805,10 +805,16 @@ Status DecodeGroup(const FrameHeader& frame_header,
       dec_state->shared->frame_dim.BlockGroupRect(group_idx), group_dec_cache,
       dec_state, first_pass));
 
-  JXL_RETURN_IF_ERROR(HWY_DYNAMIC_DISPATCH(DecodeGroupImpl)(
+  Status status = HWY_DYNAMIC_DISPATCH(DecodeGroupImpl)(
       frame_header, get_block.get(), group_dec_cache, dec_state, thread,
-      group_idx, render_pipeline_input, jpeg_data, draw));
-
+      group_idx, render_pipeline_input, jpeg_data, draw);
+  if (!status && dec_state->leniency > 0) {
+    printf("making it LF\n");
```

Comment:
:-)

Reply:
Oops, that should have been a comment, not a printf :)